go.mod

```
@@ -1,6 +1,6 @@
 module github.com/harvester/docker-machine-driver-harvester
 
-go 1.22.5
+go 1.23
```

```suggestion
go 1.23.0
```
Otherwise i'll get the error message
```
go: downloading go1.23 (linux/amd64)                                                                                                                                 
go: download go1.23 for linux/amd64: toolchain not available
```

---

Updated it.